addons/website_sale/views/templates.xml

```diff
@@ -726,29 +726,22 @@
     <template id="wizard_checkout" name="Wizard Checkout">
         <ul class="wizard pull-right hidden-xs">
             <li t-att-class="step==10 and 'text-primary' or 'text-muted'" id="step10">
-                <t t-if="step&gt;10 and step&lt;50">
-                    <a href="/shop/cart" class="text-success">
-                        Review Order<span class="chevron"></span>
-                    </a>
-                </t>
-                <t t-if="not (step&gt;10 and step&lt;50)">
-                    Review Order<span class="chevron"></span>
-                </t>
+                <t t-set="step_review_order" t-value="step&gt;10 and step&lt;50"/>
+                <a t-att-href="step_review_order and '/shop/cart' or '#'" t-att-class="step_review_order and 'text-success' or 'o_link_disable'">
+                    <span>Review Order</span>
+                </a>
+                <span class="chevron"/>
             </li>
             <li t-att-class="(step==20 and 'text-primary') or 'text-muted'" id="step20">
-                <t t-if="step&gt;20 and step&lt;50">
-                    <a href="/shop/checkout" class="text-success">
-                        <!-- billing if not loggedin - TODO -->
-                        <t>Billing</t>
-                        <t t-if="not website_sale_order or not website_sale_order.website_order_line or not website_sale_order.only_services" groups="sale.group_delivery_invoice_address"> &amp; Shipping</t>
-                        <span class="chevron"/>
-                    </a>
-                </t>
-                <t t-if="not (step&gt;20 and step&lt;50)">
-                    <t>Billing</t>
-                    <t t-if="not website_sale_order or not website_sale_order.website_order_line or not website_sale_order.only_services" groups="sale.group_delivery_invoice_address"> &amp; Shipping</t>
+                <!-- why website_sale_order is not set here ? -->
+                <t t-set="step_billing" t-value="step&gt;20 and step&lt;50"/>
+                <a t-att-href="step_billing and '/shop/checkout' or '#'" t-att-class="step_billing and 'text-success' or (step==20 and 'o_link_disable') or 'o_link_disable' ' text-muted'">
```

Comment:
Class is unreadable, if I understood it right, it will be more readable like this
```python
step_billing and 'text-success' or ('o_link_disable' + ('' if step == 20 else ' text-muted'))
```
or maybe set the class as a variable and use `t-if`/`t-else`